apps/emqx_mix_utils/lib/mix/tasks/emqx.ct.ex

```diff
@@ -45,6 +45,8 @@ defmodule Mix.Tasks.Emqx.Ct do
 
     EMQX.Mix.Utils.clear_screen()
 
+    :logger.set_primary_config(:level, :warning)
```

Comment:
Nit. OTP default is actually `notice`, `warning` is `emqx_conf`s default.

Reply:
:checkered_flag: